src/components/PropertyCard/PropertyCard.js

```diff
@@ -29,14 +77,18 @@ export default function PropertyCard({ property }) {
 				</CardContent>
 			</CardActionArea>
 
-			<CardActions disableSpacing>
-				<IconButton aria-label="add to favorites">
-					<FavoriteIcon />
+			<CardActions disableSpacing style={{ position: 'absolute', bottom: '0' }}>
+				<IconButton aria-label="add to favorites" onClick={handleLike}>
+					{like ? <FavoriteIcon fontSize="large" sx={{ color: 'red' }} /> : <FavoriteIcon fontSize="large" />}
```

Comment:
```suggestion
					<FavoriteIcon fontSize="large" sx={ like ? { color: 'red' } : {}} />
```
**THIS IS UNTESTED, PLEASE ACTUALLY TEST THIS BEFORE COMMITTING**

make it clear what the `like` is changing, just the color, instead of repeating ourselves

Reply:
This actually works! It is smart code.